client/src/main/scala/com/redi/scalademo/application/ClientModule.scala

```diff
@@ -0,0 +1,13 @@
+package com.redi.scalademo.application
+
+import com.redi.scalademo.business.NumericStringValidator
+import com.redi.scalademo.infrastructure.Client
+import com.redi.scalademo.presentation.FormListener
+
+private[application] object ClientModule {
```

Comment:
I feel like package private is overkill. That's not how we code in Java, so let's keep it simple and make it public.